xarray/testing.py

```diff
@@ -27,7 +27,7 @@ def _data_allclose_or_equiv(arr1, arr2, rtol=1e-05, atol=1e-08,
             arr1, arr2, rtol=rtol, atol=atol)
 
 
-def assert_equal(a, b):
+def assert_equal(a, b, name=''):
```

Comment:
`None` is probably a better default value. It's valid (though obviously not a great idea) to have an xarray object with name given by an empty string.